docs/standards/smart-contracts/interface-ids.md

```diff
@@ -19,9 +19,10 @@ _Interface IDs are not the most secure way to ensure that a contract implements
 | **ERC725Y**                       | `0x714df77c` | General Data key-value store.                                         |
 | **LSP1UniversalReceiver**         | `0x6bb56a14` | Universal Receiver entry function.                                    |
 | **ERC1271**                       | `0x1626ba7e` | Standard Signature Validation Method for Contracts.                   |
-| **LSP0ERC725Account**             | `0x481e0fe8` | Account that represent an identity on-chain                           |
+| **LSP0ERC725Account**             | `0x9a3bfe88` | Account that represent an identity on-chain                           |
```

Comment:
where can i verify this value?

Reply:
@Hugoo 
https://github.com/lukso-network/lsp-smart-contracts/blob/7b353b17bc16d2fc64f4d57955eba016bb79c4bb/constants.ts#L20

We also calculate them + test them to ensure that they are always correct.
If we enter them incorrectly, the CI will fail.

https://github.com/lukso-network/lsp-smart-contracts/blob/7b353b17bc16d2fc64f4d57955eba016bb79c4bb/tests/Helpers/ERC165Interfaces.test.ts#L31-L34

https://github.com/lukso-network/lsp-smart-contracts/blob/develop/contracts/Helpers/ERC165Interfaces.sol#L39-L56